api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -213,4 +254,73 @@ public void testSyncTwoFactorAuthStatus() throws Exception {
     user = userDao.findUserByEmail(EMAIL_ADDRESS);
     assertThat(user.getTwoFactorAuthCompletionTime()).isNull();
   }
+
+  @Test
+  public void testSetBypassTimes() {
+    DbUser dbUser = userDao.findUserByEmail(EMAIL_ADDRESS);
+    assertThat(dbUser.getDataUseAgreementBypassTime()).isNull();
+    assertThat(dbUser.getComplianceTrainingBypassTime()).isNull();
+    assertThat(dbUser.getBetaAccessBypassTime()).isNull();
+    assertThat(dbUser.getEraCommonsBypassTime()).isNull();
```

Comment:
are these asserts necessary? This is more testing our test code instead of our prod code.

Reply:
These are for future-proofing the assertions that come later. I want to make sure that there's only a time assigned because of the code under test, and not as a side effect of some earlier initialization logic. I'll put a little comment to that effect.